### PR TITLE
remove color inversion

### LINKED ISSUE #14893: [Feature request] Make the default night mode text color consistent with that of Desktop version
**Is your feature request related to a problem? Please describe.**

In default, text color in night mode (dark mode) is generally not consistent with the desktop version, due to the feature of comprehensive color inversion, which is unique to AnkiDroid. (Black text become white on both, though.) It seems to hinder a consistent user experience across multiple devices.

image
image
(images from https://forums.ankiweb.net/t/dark-mode-and-different-colors-of-text-in-desktop-and-android-versions/5530)

**Describe the solution you'd like**
Remove the feature of color inversion in night mode, which is unique to AnkiDroid. 
Instead of that, use the same style for text color with the desktop version, as default.

**Describe alternatives you've considered**
Make the inversion feature as an option of "Appearance" or "Accessibility" in the settings.

**Additional context**
As you know, users can even now disable the color inversion by customizing CSS as mentioned in the following:
- https://github.com/ankidroid/Anki-Android/wiki/Advanced-formatting#customize-night-mode-colors
- https://github.com/ankidroid/Anki-Android/issues/14089

This issue, however, is about whether it is appropriate to use the different color style from the desktop version as default.

### PR DESCRIPTION
**Pull request by RobozinhoD**
## Purpose / Description
Night mode wasn't consistent

## Fixes
* Fixes #14893 

## Approach
Remove night color inversion

## How Has This Been Tested?

opened cards in night mode and the colors weren't inverted

image

image

image

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by mikehardy** [comment]
If we do this, it will need a big callout in the release notes and a documentation update that inverts the explanations on how to *remove* the night mode colorization with an explanation on how to re-do it with CSS (for people that do not want change)

https://github.com/ankidroid/Anki-Android/wiki/FAQ#advanced-formatting-tips
https://github.com/ankidroid/Anki-Android/wiki/Advanced-formatting#customize-night-mode-colors

Basically, this feature is relatively polarizing as an idea (most don't care, some hate it, some probably like it) and everyone seems to have in general learned to live with it. Changing it requires some care for that reason as it is likely to be disruptive for a lot of people

**Comment by david-allison** [comment]
I don't like the feature (this breaks the canonical deck for a language in working with AND this deck was created using AnkiDroid)

We should aim to be compatible with Anki desktop, but we should do this slowly to avoid frustrating users

I believe the steps would be something similar to:

* Add a preference and default it to "off" for new users
* Define a "deprecated" preference warning which appears as a warning badge on the relevant preferences + the preference icon, which links to documentation 
* Block new translations
* Define a "deprecated" preference error: if the preference is still enabled, inform the user that it has no effect and link to the documentation
  * Figure out the UX here, we may want a user to be able to see how the preference changes the card whilst disabling functionality
* If disabled, remove the preference + key from the collection
* If enabled, keep the preference until n+1 and the preference was disabled
* If never disabled, wait a few versions then remove the preference
  * Maybe have a "breaking changes" doc

**Comment by BrayanDSO** [comment]
The layers of filter for frustrated people would be the people that:
- Are a previous user
- Use night mode
- Don't use the desktop version and this way won't discover the incosistence
- Don't know CSS
- Have downloaded a custom deck (since they don't know CSS to add a custom color themselves)
    - which doesn't have `.night_mode`/`.nightMode`
    - which has custom colors
        - that display well when inverted and don't when not

So I don't think that the number of affected people will be big and actually expect more people being frustrated because the color is inverted than because it is not inverted. Therefore, I prefer being disruptive and doing it at once (but I won't quote Zuckerberg motto here).

The cautious approach is fine too, but it's up to the people that want to carry it given the limited human resources we have and the amount of stuff that is more important than this

**Comment by mikehardy** [comment]
I think disrupt and "fix" with altered documentation / changelog mention is okay
the alternative is a whole lot of work for likely little return yes

**Comment by david-allison** [comment]
Disagree & commit: let's do this!

I'm going to be working on deprecation in preferences for `ReadText`, if this gets in any time soon, we have the option, but I suspect it won't be necessary

**Review by BrayanDSO — APPROVED** [review]
LGTM

**Review by david-allison — APPROVED** [review]
Also LGTM, pending CI
